src/core/agent/services/multiSigService.ts

```diff
@@ -333,7 +333,7 @@ class MultiSigService extends AgentService {
     const smids = icpMsg[0].exn.a.smids;
     // @TODO - foconnor: These searches should be optimised, revisit.
     const ourIdentifiers = await Agent.agent.identifiers.getIdentifiers();
-    const ourConnections = await Agent.agent.connections.getConnections();
+    const ourConnections = await Agent.agent.connections.getGroupConnections();
```

Comment:
Good catch! I also think we can improve without introducing this new function though:
1. Find `ourIdentifier` first and extract the `groupId`
2. Call `getMultisigLinkedContacts(groupId)

Will this work?

Reply:
I have updated it!